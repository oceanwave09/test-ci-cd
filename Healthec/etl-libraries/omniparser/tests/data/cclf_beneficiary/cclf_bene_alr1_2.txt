"BENE_MBI_ID","BENE_HIC_NUM","BENE_1ST_NAME","BENE_LAST_NAME","BENE_SEX_CD","BENE_BRTH_DT","BENE_DEATH_DT","MASTER_ID","B_EM_LINE_CNT_T"
"1AAAAMKR10","","2501FirstName","2501LastName","2","01/01/1965","","202084193","6"
"1AAAAMKR20","","2502FirstName","2502LastName","2","01/01/1947","","383593303","2"
"1AAAAMKR30","","2503FirstName","2503LastName","1","01/01/1953","","382386354","16"
"1AAAAMKR40","","2504FirstName","2504LastName","2","01/01/1952","","383593303","4"
