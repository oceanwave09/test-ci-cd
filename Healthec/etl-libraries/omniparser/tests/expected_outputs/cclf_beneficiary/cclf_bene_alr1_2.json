[
	{
		"b_em_line_cnt_t": "6",
		"bene_1st_name": "2501FirstName",
		"bene_brth_dt": "01/01/1965",
		"bene_last_name": "2501LastName",
		"bene_mbi_id": "1AAAAMKR10",
		"bene_sex_cd": "2",
		"master_id": "202084193",
		"row_id": "c427dbd1-cc9d-4365-b44a-3359d44c3db8"
	},
	{
		"b_em_line_cnt_t": "2",
		"bene_1st_name": "2502FirstName",
		"bene_brth_dt": "01/01/1947",
		"bene_last_name": "2502LastName",
		"bene_mbi_id": "1AAAAMKR20",
		"bene_sex_cd": "2",
		"master_id": "383593303",
		"row_id": "471f8aca-fa98-4468-84c5-e4e1f13a34eb"
	},
	{
		"b_em_line_cnt_t": "16",
		"bene_1st_name": "2503FirstName",
		"bene_brth_dt": "01/01/1953",
		"bene_last_name": "2503LastName",
		"bene_mbi_id": "1AAAAMKR30",
		"bene_sex_cd": "1",
		"master_id": "382386354",
		"row_id": "00a65bbd-cfff-45bb-9a8c-62fe5ba1a953"
	},
	{
		"b_em_line_cnt_t": "4",
		"bene_1st_name": "2504FirstName",
		"bene_brth_dt": "01/01/1952",
		"bene_last_name": "2504LastName",
		"bene_mbi_id": "1AAAAMKR40",
		"bene_sex_cd": "2",
		"master_id": "383593303",
		"row_id": "c7ba9525-50ab-4d0b-a796-000e98de790d"
	}
]
